guides/debug-logs.md

```diff
@@ -0,0 +1,42 @@
+# Debug Logs
+
+Many Cypress packages use the [`debug` module][debug] to log runtime info to the console.
+
+## Choosing a namespace
+
+The naming scheme for debug namespaces should generally follow this pattern:
+
+```
+cypress:{packageName}:{relative path to file from src root, using : to separate directories, minus index if applicable}
```

Comment:
I wonder if in the future we can do this automatically, similar to what we do [in the dashboard](https://github.com/cypress-io/cypress-services/blob/9d8899e5c68f2221f81b0a5a79bf1e659d313d71/packages/debug/src/debug.ts#L21-L37)

```
import { makeDebug } from '@packages/debug'
const debug = makeDebug(__filename)
```

Reply:
Yeah, I would love to do this in this repo too so the burden isn't on individual contributors and reviewers to know these guidelines.